Tools/PIDML/CMakeLists.txt

```diff
@@ -36,3 +36,10 @@ o2physics_add_dpl_workflow(qa-pid-ml
                   SOURCES qaPidML.cxx
                   PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore ONNXRuntime::ONNXRuntime
                   COMPONENT_NAME Analysis)
+
+o2physics_add_dpl_workflow(kaon_pid
```

Comment:
As the GitHub says - you need to use spaces instead of tabs.

Consider clang-format to automatize code formatting: https://github.com/AliceO2Group/CodingGuidelines